client/src/components/Video.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import "./Video.css";
+import { AiOutlineLike, AiOutlineDislike, AiOutlineDelete } from 'react-icons/ai';
+
+const Video = ({ id, title, url, rating, onDelete }) => {
+  
+  const [LiveRating, setLiveRating] = useState(rating); 
```

Comment:
`LiveRating` should start with a lower case, we use camel case for variables and they start with a lower case 😃